cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -528,6 +528,10 @@ private class ServiceImpl(
                         ccpaConsentStatus = csd.ccpa
                         ccpaUuid = csd.ccpa?.uuid
                         ccpaDateCreated = csd.ccpa?.dateCreated
+
```

Comment:
Before merging, please remember to remove those logs because they will still log data in the release artifact

Reply:
sure